conda-recipes/tensorflow-datasets-feedstock/ci_support/run_docker_build.sh

```diff
@@ -50,7 +50,7 @@ docker run ${DOCKER_RUN_ARGS} \
                         -e HOST_USER_ID \
                         -e UPLOAD_PACKAGES \
                         -e CI \
-                        -a stdin -a stdout -a stderr \
+                        -a stdin -a stdout -a stderr  --privileged=true -u root \
```

Comment:
Interesting how the travis failure mentions tty. Can you try undoing the TTY change we made last time for the anvil container?

Reply:
I tried that too and the old error appears "input device is not tty"

Reply:
Removing --privileged=true worked.